dali/operators/random/uniform_distribution_cpu.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/random/rng_base_cpu.h"
+#include "dali/operators/random/uniform_distribution.h"
+#include "dali/pipeline/operator/arg_helper.h"
+
+namespace dali {
+
+DALI_SCHEMA(random__Uniform)
```

Comment:
Why do we need the double-underscore here? Looks awkward :-|

Reply:
This is the way we create modules. `__` is the module separator (can't use `.` here). The operator is actually `random.Uniform`